docs/ciderEditor.md

```diff
@@ -0,0 +1,25 @@
+# The CIDER (Contenteditable-Input-Diff-Error-Reconcile) editor
```

Comment:
Hmm, I feel like docs should be wrapped to the same line length as code for easy reading in any editor, but we seem to be a bit all over the place with doc line lengths... 😭